### PR TITLE
Adding pymispwarninglists

### PR DESCRIPTION
**Pull request by yangdan4**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml"
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [x] Source is from official source
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged)
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details)
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

### CODE REVIEW

**Comment by yangdan4** [comment]
Willing to be listed

**Comment by yangdan4** [comment]
@conda-forge-admin, please ping team @conda-forge/help-python
for review, thank you!

File: recipes/pymispwarninglists/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
+  sha256: "c60eb6fecf415a4e122af92c4c22ab14553a52fb02c8e6e0809ce1e8adc05a47"
+
+build:
```
Comment (synapticarbors) [inline recipes/pymispwarninglists/meta.yaml:12]: I believe this package can be `noarch: python`. See:

https://conda-forge.org/docs/maintainer/knowledge_base.html#noarch-python

File: recipes/pymispwarninglists/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+    - python
+    - poetry
+
+test:
+  imports:
+    - pymispwarninglists
+    - pymispwarninglists.data.misp-warninglists.tools
+
+about:
+  home: "The package home page"
```
Comment (synapticarbors) [inline recipes/pymispwarninglists/meta.yaml:33]: `home: https://github.com/MISP/PyMISPWarningLists`

File: recipes/pymispwarninglists/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+  license: "3BSD"
```
Comment (synapticarbors) [inline recipes/pymispwarninglists/meta.yaml:34]: `license: BSD-3-Clause`

File: recipes/pymispwarninglists/meta.yaml [hunk 4]
```diff
@@ -0,0 +1,43 @@
...
+  license_family: "BSD"
+  license_file: "LICENSE.txt"
+  summary: "Python API for the MISP warning lists."
+  doc_url: ""
+  dev_url: ""
```
Comment (synapticarbors) [inline recipes/pymispwarninglists/meta.yaml:39]: `dev_url: https://github.com/MISP/PyMISPWarningLists`

File: recipes/pymispwarninglists/meta.yaml [hunk 5]
```diff
@@ -0,0 +1,43 @@
...
+    - pymispwarninglists
+    - pymispwarninglists.data.misp-warninglists.tools
+
+about:
+  home: "The package home page"
+  license: "3BSD"
+  license_family: "BSD"
+  license_file: "LICENSE.txt"
+  summary: "Python API for the MISP warning lists."
+  doc_url: ""
```
Comment (synapticarbors) [inline recipes/pymispwarninglists/meta.yaml:38]: delete `doc_url` entry

File: recipes/pymispwarninglists/meta.yaml [hunk 6]
```diff
@@ -0,0 +1,43 @@
...
+  dev_url: ""
+
+extra:
+  recipe-maintainers:
+    - your-github-id-here
```
Comment (synapticarbors) [inline recipes/pymispwarninglists/meta.yaml:43]: you need to change this to your github user name

File: recipes/pymispwarninglists/meta.yaml [hunk 7]
```diff
@@ -0,0 +1,43 @@
...
+test:
+  imports:
+    - pymispwarninglists
+    - pymispwarninglists.data.misp-warninglists.tools
+
+about:
+  home: "The package home page"
+  license: "3BSD"
+  license_family: "BSD"
+  license_file: "LICENSE.txt"
```
Comment (synapticarbors) [inline recipes/pymispwarninglists/meta.yaml:36]: `license_file: LICENSE`

**Comment by jakirkham** [comment]
Any updates here, @yangdan4? 🙂

**Pull request closed without merging**